src/main/osd/osd.h

```diff
@@ -159,6 +159,7 @@ typedef enum {
     OSD_CAMERA_FRAME,
     OSD_EFFICIENCY,
     OSD_TOTAL_FLIGHTS,
+    OSD_TOTAL_PACKS,
```

Comment:
// *** IMPORTANT ***
// Whenever new elements are added to 'osd_items_e', make sure to increment
// the parameter group version for 'osdConfig' in 'osd.c'